lib/cdo/lesson_import_helper.rb

```diff
@@ -20,12 +20,12 @@ module LessonImportHelper
   # @param [Lesson] lesson - Code Studio Lesson object to update.
   # @param [Hash] cb_lesson_data - Lesson and activity data to import.
   def self.update_lesson(lesson, cb_lesson_data = {})
-    # In the future, only levelbuilder should be added to this list.
-    raise unless [:development, :adhoc].include? rack_env
+    raise unless [:development, :adhoc, :levelbuilder].include? rack_env
+    raise if lesson.script.is_stable
```

Comment:
just noting that this means we won't be able to re-import the 20-21 scripts in development or adhoc after this, which is fine because we can/should be importing the 21-22 scripts now that they are available.